e2e-tests/src/test/java/gov/nasa/jpl/aerie/e2e/ConstraintsTests.java

```diff
@@ -454,13 +478,15 @@ void compilationFailsOutdatedSimulationSimDatasetId() throws IOException {
       final int newSimDatasetId = hasura.awaitSimulation(planId).simDatasetId();
       // This test causes the endpoint to throw an exception when it fails to compile the constraint,
       // as it cannot find the external dataset resource in the set of known resource types.
-      final var constraintResponses = hasura.checkConstraints(planId);
+      final var constraintResponses = hasura.checkConstraints(planId, newSimDatasetId);
       assertEquals(1,constraintResponses.size());
       final var constraintRepsonse = constraintResponses.get(0);
```

Comment:
Minor: typo
```suggestion
      final var constraintResponse = constraintResponses.get(0);
```

Reply:
noooooooo!!!!!! stupid fatfingers. I fixed it.